jiant/tasks/lib/templates/hacky_tokenization_matching.py

```diff
@@ -31,72 +30,12 @@ def input_flat_strip(tokens):
     return "".join(tokens).lower()
 
 
-def delegate_flat_strip(tokens, tokenizer, return_indices=False):
-    if isinstance(tokenizer, transformers.BertTokenizer):
-        return bert_flat_strip(tokens=tokens, return_indices=return_indices)
-    elif isinstance(tokenizer, transformers.RobertaTokenizer):
-        return roberta_flat_strip(tokens=tokens, return_indices=return_indices)
-    elif isinstance(tokenizer, transformers.AlbertTokenizer):
-        return albert_flat_strip(tokens=tokens, return_indices=return_indices)
-    elif isinstance(tokenizer, transformers.XLMRobertaTokenizer):
-        return xlm_roberta_flat_strip(tokens=tokens, return_indices=return_indices)
-    else:
-        raise KeyError(type(tokenizer))
-
-
-def bert_flat_strip(tokens, return_indices=False):
-    ls = []
-    count = 0
-    indices = []
-    for token in tokens:
-        if token.startswith("##"):
-            token = token.replace("##", "")
-        else:
-            pass
-        ls.append(token)
-        indices += [count] * len(token)
-        count += 1
-    string = "".join(ls).lower()
-    if return_indices:
-        return string, indices
-    else:
-        return string
-
-
-def roberta_flat_strip(tokens, return_indices=False):
-    ls = []
-    count = 0
-    indices = []
-    for token in tokens:
-        if token.startswith("Ġ"):
-            token = token.replace("Ġ", "")
-        else:
-            pass
-        ls.append(token)
-        indices += [count] * len(token)
-        count += 1
-    string = "".join(ls).lower()
-    if return_indices:
-        return string, indices
-    else:
-        return string
-
-
-def xlm_roberta_flat_strip(tokens, return_indices=False):
-    # TODO: Refactor to use general SentencePiece function  (issue #1181)
-    return albert_flat_strip(tokens=tokens, return_indices=return_indices)
-
-
-def albert_flat_strip(tokens, return_indices=False):
+def flat_strip(tokens, tokenizer, return_indices=False):
```

Comment:
Do we have tests for this?

Reply:
Not yet. I'll add some now

Reply:
We did not have tests before but I just added tests here: https://github.com/nyu-mll/jiant/pull/1289. I'll check that the added tests pass on this PR.

Reply:
Thanks for requesting the tests! Found a bug with my implementation.